pkg/collector/corechecks/embed/process_agent.go

```diff
@@ -131,7 +131,7 @@ func (c *ProcessAgentCheck) run() error {
 func (c *ProcessAgentCheck) Configure(data integration.Data, initConfig integration.Data, source string) error {
 	// only log whether process check is enabled or not but don't return early, because we still need to initialize "binPath", "source" and
 	// start up process-agent. Ultimately it's up to process-agent to decide whether to run or not based on the config
-	if enabled := config.Datadog.GetBool("process_config.enabled"); !enabled {
```

Comment:
Are we currently logging this message if only container collection is enabled?

Reply:
You're right. I thought that the log should be printed when process-agent is not enabled but in this case it seems that it refers only to the `process` check. I think that it's ok to keep it as it is and we can revisit it later.